tracer/src/Datadog.Trace.Trimming/build/Datadog.Trace.Trimming.xml

```diff
@@ -70,9 +70,14 @@
    <assembly fullname="Microsoft.Extensions.Logging" />
    <assembly fullname="Microsoft.Extensions.Logging.Abstractions" />
    <assembly fullname="Microsoft.Extensions.Primitives">
+      <type fullname="Microsoft.Extensions.Primitives.StringSegment" />
```

Comment:
TIL we can use `StringSegment` in .NET Core without an external reference!

Reply:
I was a little surprised, but this change was automatically generated, so I left it.